### PR TITLE
Packages READMEs and rules

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

File: PACKAGES.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+- `@plone/registry`
+- `@plone/client`
+- `@plone/components`
+
+### Rules
+
+They can't depend on any other `@plone/*` package, with only one exception: `@plone/types`.
+They must be published and bundled in a traditional (transpiled) way.
+The bundle of these packages must work on both commonjs and ESM environments.
```
Comment (stevepiercy) [inline PACKAGES.md]: What is ESM? "European Stability Mechanism"? The first time an acronym is used, you must spell out its proper name, and optionally follow it with its acronym in parentheses.

European Stability Mechanism (ESM)

**Review by stevepiercy — CHANGES_REQUESTED** [review]
I reviewed docs.

- One question to define "ESM"
- Add new files to CI https://github.com/plone/volto/blob/main/.github/workflows/readme-link-check.yml#L23

File: PACKAGES.md (hunk 1 above)
Comment (sneridagh) [reply PACKAGES.md]: ECMAScript Module (ESM):

https://medium.com/@halilatilla/differences-between-javascript-modules-cjs-amd-umd-and-esm-f60124de131b#:~:text=ECMAScript%20Module%20(ESM)%3A,static%20analysis%20and%20tree%20shaking.

**Comment by sneridagh** [comment]
@stevepiercy btw, the extra carriage returns that you are adding, VSCode complains about them:

```
MD012/no-multiple-blanks: Multiple consecutive blank lines [Expected: 1; Actual: 2]markdownlint[MD012](https://github.com/DavidAnson/markdownlint/blob/v0.33.0/doc/md012.md)
```

**Comment by sneridagh** [comment]
@stevepiercy same for spaces after the list dashes: 

```
MD030/list-marker-space: Spaces after list markers [Expected: 1; Actual: 3]markdownlint[MD030](https://github.com/DavidAnson/markdownlint/blob/v0.33.0/doc/md030.md)
```

**Comment by stevepiercy** [comment]
I use MyST primarily, where 4 spaces helps with proper indentation of nested lists, code blocks, and admonitions. There is no standard for Markdown flavors, and we don't require linting Markdown, CommonMark, or GitHub's flavor of Markdown anywhere.

**Review by stevepiercy — APPROVED** [review]

**Comment by sneridagh** [comment]
@stevepiercy merging for now, we will continue adding and improving, I'd like to have something there.

**Comment by sneridagh** [comment]
@stevepiercy I had this extension, I removed it from my setup: https://github.com/DavidAnson/vscode-markdownlint

**Comment by stevepiercy** [comment]
@sneridagh I assume you could configure it to silence specific warnings. If it helps you, then use it. I always use my editor's preview feature or build docs to catch mistakes.

I have an ingrained habit of using the `tab` key to indent, and configure Markdown/MyST/reST to indent 4 spaces:

```
-<TAB>Bullet List Item
```

or

```
1.<TAB>Enumerated List Item
```

or

```
Definition List Term<RETURN>
:<TAB>Definition List Definition
```

This way I always have consistent indentation of any type of list or nested items.

**Pull request merged**